app/code/core/Mage/Adminhtml/Block/System/Config/Form/Field.php

```diff
@@ -83,10 +83,10 @@ public function render(Varien_Data_Form_Element_Abstract $element)
                 $defTextArr = [];
                 foreach ($options as $k => $v) {
                     if ($isMultiple) {
-                        if (is_array($v['value']) && in_array($k, $v['value'])) {
+                        if (array_key_exists("value", $v) && is_array($v['value']) && in_array($k, $v['value'])) {
```

Comment:
The rest of code uses single quotes, I suggest to keep it up, so change it to `array_key_exists('value', $v)`.